### PR TITLE
[Hot fix]Fix parse_func and fault_handler of ModelWrapper

### PR DESCRIPTION
**Pull request by pan-x-c**
## Description

As the title says

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

**Review by DavdGao — COMMENTED** [review]
Please see inline comments.

File: src/agentscope/models/model.py [hunk 1]
```diff
@@ -107,6 +107,15 @@ def raw(self) -> dict:
         """Raw dictionary field."""
         return self._raw
 
+    def __str__(self) -> str:
+        serialized_fields = {
+            "text": self.text,
+            "embedding": self.embedding,
+            "image_urls": self.image_urls,
```
[the remark this answers is not in the document]
Comment (pan-x-c) [reply src/agentscope/models/model.py:114]: It is necessary to use `image` as the prefix (For situations where a model returns multiple URLs including audio and image at the same time)

File: src/agentscope/agents/dict_dialog_agent.py [hunk 2]
```diff
@@ -7,10 +7,21 @@
...
 
 
+def parse_dict(response: ModelResponse) -> ModelResponse:
+    """Parse function for DictDialogAgent"""
+    return ModelResponse(raw=json.loads(response.text))
+
+
+def default_response(response: ModelResponse) -> ModelResponse:
+    """The default response of fault_handler"""
+    return ModelResponse(raw={"speak": response.text})
```
[the remark this answers is not in the document]
Comment (pan-x-c) [reply src/agentscope/agents/dict_dialog_agent.py:22]: same as above

File: src/agentscope/agents/dict_dialog_agent.py [hunk 3]
```diff
@@ -129,7 +142,7 @@ def reply(self, x: dict = None) -> dict:
             parse_func=self.parse_func,
             fault_handler=self.fault_handler,
             max_retries=self.max_retries,
-        ).text
+        ).raw
```
[the remark this answers is not in the document]
Comment (pan-x-c) [reply src/agentscope/agents/dict_dialog_agent.py:145]: In DictDialogAgent, the response is in `dict` format, while the text field only accepts `str`

**Review by DavdGao — APPROVED** [review]
LGTM

**Pull request merged**